craco.config.js

```diff
@@ -0,0 +1,16 @@
+// craco.config.js
+const path = require(`path`);
+const alias = require(`./src/config/aliases`);
+
+const SRC = `./src`;
```

Comment:
Please use one terminology. change 'SRC' to 'src'

Reply:
done